pdns/recursordist/docs/settings.rst

```diff
@@ -419,8 +419,11 @@ found, the recursor fallbacks to sending 127.0.0.1.
 
 ``edns-outgoing-bufsize``
 -------------------------
+.. versionchanged:: 4.2.0
+  Before 4.2.0, the default was 1680
+
 -  Integer
--  Default: 1680
+-  Default: 1232
 
 This is the value set for the EDNS0 buffer size in outgoing packets.
 Lower this if you experience timeouts.
```

Comment:
It would be nice to document where the `1232` value is coming from, so we don't have to guess 10 years from now ;)